QuizApp/QuizApp/UI/Quiz/QuizViewController.swift

```diff
@@ -167,6 +180,52 @@ extension QuizViewController: UICollectionViewDataSource {
         return cell
     }
 
+    func collectionView(
+        _ collectionView: UICollectionView,
+        layout collectionViewLayout: UICollectionViewLayout,
+        referenceSizeForHeaderInSection section: Int
+    ) -> CGSize {
+        quizes.keys.count < 2 ? CGSize(width: 0, height: 0) : CGSize(width: 0, height: 40)
```

Comment:
Nice! You could simplify:

`quizes.keys.count < 2 ? .zero : CGSize(width: 0, height: 40)`